bin/msstats.R

```diff
@@ -10,18 +10,23 @@ library(MSstats, warn.conflicts = FALSE)
 # Convert EncyclopeDIA results to a format for MSstats
 encyclopediaToMsstats <- function(peptides_txt, proteins_txt) {
   id_vars <- c("Peptide", "Protein", "numFragments")
-  proteins <- read.table(proteins_txt,
+  prot2pep <- read.table(proteins_txt,
                          sep = "\t",
                          header = TRUE,
                          stringsAsFactors = TRUE,
-                         check.names = FALSE)
+                         check.names = FALSE) %>%
+    select(Protein, PeptideSequences) %>%
+    mutate(Peptide = str_split(PeptideSequences, ";")) %>%
+    select(Protein, Peptide) %>%
+    unnest(Peptide)
 
   df <- read.table(peptides_txt,
                    sep = "\t",
                    header = TRUE,
                    stringsAsFactors = FALSE,
                    check.names = FALSE) %>%
-    inner_join(proteins["Protein"], by = "Protein") %>%
+    select(-Protein) %>%
+    inner_join(prot2pep, by = "Peptide") %>%
```

Comment:
it might be wise to add here the `unmatched ="error"` argument https://dplyr.tidyverse.org/reference/mutate-joins.html just to raise a red flag if this error re-surges

Reply:
It's difficult, because is reality we don't expect a perfect match. The Proteins accepted at 1% FDR *should* be a subset of the peptides detected at 1% FDR. However, it is theoretically possible for a protein to be accepted despite none of its peptides being accepted at 1% FDR.

Reply:
what about .... adding a `stopifnot` that checks that at least 95% (or 90 ... or 75 ... or 50 ... whatever seems like a good 'red flag ammount') of the stuff matched? (I think that would be a reasonable assumption)

Reply:
That seems like a good idea. I'll add it 🤔

Reply:
LGTM